app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,12 @@
 class SessionsController < ApplicationController
   skip_before_action :require_login
 
+  def getList
```

Comment:
@JuanMaGiraldo ruby does not use camelcase for defining methods or variables?

Reply:
I realized later that I was using camel case in place of snake case. I fixed the name to list_inputs to sound like attribute.